src/Kris/LaravelFormBuilder/FormHelper.php

```diff
@@ -456,6 +456,10 @@ public function appendMessagesWithPrefix(MessageBag $messageBag, $prefix, array
      */
     public function transformToDotSyntax($string)
     {
+        if (!$string) {
```

Comment:
That also triggers for `0` and `'0'`. Probably never happens, but still.

Reply:
Oh, I followed the code style from line 277 but yeah better to check with is_null method.